packages/xo-web/src/common/xo/index.js

```diff
@@ -564,7 +564,11 @@ subscribeVolumeInfo.forceRefresh = (() => {
   }
 })()
 
-export const subscribeSrsUnhealthyVdiChainsLength = createSubscription(() => _call('sr.getAllUnhealthyVdiChainsLength'))
+export const subscribeSrsUnhealthyVdiChainsLength = createSubscription(() => {
+  const _isAdmin = isAdmin(store.getState())
+
+  return _isAdmin ? _call('sr.getAllUnhealthyVdiChainsLength') : undefined
+})
```

Comment:
I'm not fond of permission handling being done in multiple places: if the logic change in `xo-server` it would be nice if it would work without having to update `xo-web`.

But I recognized that this PR addresses error messages in the logs due to not enough permissions, and I don't see an easy way to achieve that, but we'll have to  find better ways to do this in XO 6 :slightly_smiling_face: